Hymns/Favorites/FavoritesView.swift

```diff
@@ -0,0 +1,33 @@
+import SwiftUI
+
+struct FavoritesView: View {
+    @ObservedObject private var viewModel = FavoritesViewModel()
+    var allHymns: [DummyHymnView] = testData
+
+    var body: some View {
+        VStack {
+            CustomTitle(title: "Favorites")
+            List {
+                ForEach(viewModel.itemEntities) { (itemEntity: FavoritedEntity) in
+                    if itemEntity.isInvalidated {
+                        EmptyView()
```

Comment:
What does an EmptyView look like? Will it just have a bunch of empty slots in the list?

Reply:
No, thats the great thing. Empty view doesn't have a gap at all. :) Works very well with ForEach or List